plugins/callback/log_plays.py

```diff
@@ -79,30 +80,43 @@ def log(self, host, category, data):
                 if invocation is not None:
                     data = json.dumps(invocation) + " => %s " % data
 
-        path = os.path.join(self.log_folder, host)
+        path = os.path.join(self.log_folder, result._host.get_name)
```

Comment:
Isn't it a function?
```suggestion
        path = os.path.join(self.log_folder, result._host.get_name())
```

Reply:
Damn, what a shame =/
Fixed.